content/fr/_index.html

```diff
@@ -47,12 +47,12 @@ <h2>Les défis de la migration de plus de 150 microservices vers Kubernetes</h2>
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2019" button id="desktopKCButton">Venez au KubeCon Barcelone du 20 au 23 mai 2019</a>
+        <a href="https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2019" button id="desktopKCButton">Venez au KubeCon Shanghai du 24 au 26 Juin 2019</a>
         <br>
         <br>
         <br>
         <br>
-        <a href="https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2019" button id="desktopKCButton">Venez au KubeCon Shanghai du 24 au 26 juin 2019</a>
+        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019" button id="desktopKCButton">Venez au KubeCon San Diego du 18 au 21 Novembre 2019</a>
```

Comment:
```suggestion
        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019" button id="desktopKCButton">Venez au KubeCon San Diego du 18 au 21 Nov 2019</a>
```

Je propose de mettre `Nov` pour reste sur la même taille du bouton (cf. screenshot https://d.pr/free/i/C0iurD)

Si non, ça va déborder un peu (screenshot https://d.pr/free/i/YXot4M)